include/powsybl/math/UndirectedGraph.hxx

```diff
@@ -38,6 +38,18 @@ void UndirectedGraph<V, E>::checkVertex(unsigned long v) const {
     }
 }
 
+template <typename V, typename E>
+void UndirectedGraph<V, E>::cleanVertices(unsigned long v) {
```

Comment:
I think this method is not well placed. The `checkVertex` and `checkEdge` are at the top of the file cause they are used everywhere, and you need the implementation before calling them. The cleanVertices is used only in removeVertex, so it's OK to move this method.

Reply:
>The checkVertex and checkEdge are at the top of the file cause they are used everywhere

Strange! Without using a hxx file, I would understand but here I don't. I moved them and its ok.